discovery-provider/src/utils/elasticdsl.py

```diff
@@ -44,17 +50,37 @@ def hits_by_id(found):
 
 
 def popuate_user_metadata_es(user, current_user):
-    user_following = user.get("following_ids", [])
-    current_user_following = current_user.get("following_ids", [])
-    user["does_current_user_follow"] = user["user_id"] in current_user_following
-    user["does_follow_current_user"] = current_user["user_id"] in user_following
+    user["total_balance"] = str(
+        int(user.get("balance", "0") or "0")
+        + int(user.get("associated_wallets_balance", "0") or "0")
+        + to_wei(user.get("associated_sol_wallets_balance", "0") or 0)
+        + to_wei(user.get("waudio", "0") or 0)
```

Comment:
I believe we need to multiply the solana wormhole audio by 10^10

Reply:
i think that's what `to_wei` does right? similar thing happens here https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/src/queries/get_balances.py#L69

Reply:
this looks right. @sddioulde can you confirm for us?